plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/version/KubernetesVersionManagerImpl.java

```diff
@@ -331,12 +345,14 @@ public KubernetesSupportedVersionResponse addKubernetesSupportedVersion(final Ad
 
         KubernetesSupportedVersionVO supportedVersionVO = new KubernetesSupportedVersionVO(name, semanticVersion, template.getId(), zoneId, minimumCpu, minimumRamSize);
         supportedVersionVO = kubernetesSupportedVersionDao.persist(supportedVersionVO);
+        CallContext.current().putContextParameter(KubernetesSupportedVersion.class, supportedVersionVO.getUuid());
 
         return createKubernetesSupportedVersionResponse(supportedVersionVO);
     }
 
     @Override
-    @ActionEvent(eventType = KubernetesVersionEventTypes.EVENT_KUBERNETES_VERSION_DELETE, eventDescription = "Deleting Kubernetes supported version", async = true)
+    @ActionEvent(eventType = KubernetesVersionEventTypes.EVENT_KUBERNETES_VERSION_DELETE,
+            eventDescription = "deleting Kubernetes supported version", async = true)
```

Comment:
```suggestion
            eventDescription = "Deleting Kubernetes supported version", async = true)
```

Reply:
We have some code in ActionEventUtils or Interprator which prefixes `Successfully completed` to the event description